### PR TITLE
loosen detection of null island records to those in the general vicinity

### PR DESCRIPTION
**Pull request by missinglink**
loosen detection of null island records to those in the general vicinity

fix for https://github.com/openaddresses/openaddresses/issues/5008

### CODE REVIEW

**Comment by orangejulius** [comment]
Looks like a good thing to have, we were bound to need it some day. I wonder how records end up _not quite_ at 0,0 though :shrug:

File: test/isValidCsvRecord.js [hunk 1]
```diff
@@ -156,14 +156,40 @@ tape('record with lon/lat parseable as 0/non-0 should return true', test => {
...
+tape('record with lon/lat very close to 0,0 should return false', test => {
   const record = {
     LON: '0.000001',
     LAT: '0.000000',
     NUMBER: 'Number',
     STREET: 'Street'
   };
 
-  test.ok( isValidCsvRecord(record), 'should be accepted');
+  test.notOk(isValidCsvRecord(record), 'should be accepted');
```
[the remark this answers is not in the document]
Comment (missinglink) [reply test/isValidCsvRecord.js]: resolved via rebase

**Comment by missinglink** [comment]
> I wonder how records end up not quite at 0,0 though

only 13 thankfully!
https://pelias.github.io/compare/#/v1/reverse?size=40&point.lat=0&point.lon=0&debug=0

**Comment by missinglink** [comment]
Actually 13 is a very respectable number, seems the tropical paradise of Null Island is suffer from the travel ban 😆

**Comment by missinglink** [comment]
For anyone who's interested, the value I chose (0.0005) ends up being an extremely tiny buffer which I would be ok with widening at a later day if required since there is literally nothing there except a weather buoy and some lost & confused cartographers.

Distance from (0.0005,0.0005) -> (0,0) == 78.5 m (257.4 ft)
Distance from (0.0005,0.0005) -> (-0.0005,-0.0005) == 156.9 m (514.8 ft)

**Pull request merged**